controllers/resources/infinispan.xmltmpl

```diff
@@ -84,12 +60,9 @@
         <MERGE3 min_interval="1000" max_interval="3000" check_interval="5000" stack.combine="COMBINE"/>
         {{ end }}     
     </stack>
-    <stack name="xsite" extends="image-{{ .JGroups.Transport }}">
-        <relay.RELAY2 xmlns="urn:org:jgroups" site="{{ .XSite.Name }}" max_site_masters="{{ .XSite.MaxRelayNodes }}"
-                      can_become_site_master="{{ .XSite.RelayNodeCandidate }}"/>
-        <remote-sites default-stack="relay-{{ .XSite.Transport }}">
-            <remote-site name="{{ .XSite.Name }}"/>
-            {{ range $it := .XSite.Backups }}
+    <stack name="xsite" extends="image-tcp">
```

Comment:
this line wasn't changed by me.

Reply:
actually, you can revert the changes in this file since https://github.com/infinispan/infinispan-operator/commit/5df1047a5f6825e4c9d7692d96b7537a21cdedeb is not merged in this branch.

Reply:
Updated :+1: